src/VirtoCommerce.NotificationsModule.Core/Model/Notification.cs

```diff
@@ -10,14 +9,15 @@ namespace VirtoCommerce.NotificationsModule.Core.Model
     /// <summary>
     /// Base class for Notification
     /// </summary>
-    public abstract class Notification : AuditableEntity, ICloneable
+    public abstract class Notification : AuditableEntity, ICloneable, IHasOuterId
     {
         /// <summary>
         /// For detecting owner
         /// </summary>
         public TenantIdentity TenantIdentity { get; set; } = TenantIdentity.Empty;
         public bool IsActive { get; set; }
         public string LanguageCode { get; set; }
+        public string Alias { get; set; }
```

Comment:
Add comments for this property. e.g `This field represents an alias for the notification type and is used only for backward compatibility with old notification names that are stored and used by API clients.`